### PR TITLE
add env and changed logging and put it in order

### PR DESCRIPTION
**Pull request by yaroslavvasilenko**

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -1,42 +1,41 @@
...
-
-	ctx = logging.ContextWithLogger(ctx, logger)
+	cfg, err := config.GetConf()
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.Info("config initializing")
 
-	app, err := app.NewApp()
+	a, err := app.NewApp(cfg)
```
Comment (silverCase) [inline cmd/main.go:18]: Name variables so people can easily read code.
e.g. a -> app

File: cmd/main.go [hunk 2]
```diff
@@ -1,42 +1,41 @@
...
 	if err != nil {
-		logger.Fatal("fatal init")
+		log.Fatal(err)
 	}
 
-	for update := range *app.BUpdateChannel {
+	log.Info("Running Application")
```
Comment (silverCase) [inline cmd/main.go:23]: Make logging message more concrete: 
"Running Application" -> "Update Channel created"/"Bot polling started" etc

File: cmd/main.go [hunk 3]
```diff
@@ -1,42 +1,41 @@
...
 
 			msg := tgbotapi.NewMessage(update.Message.Chat.ID, update.Message.Text)
 			msg.ReplyToMessageID = update.Message.MessageID
 
-			_, err := app.Bapi.Send(msg)
+			m, err := a.Bot.Send(msg)
 			if err != nil {
-				logger.Debug("deb")
-				return
+				log.Info(m)
```
Comment (silverCase) [inline cmd/main.go:34]: Rework logging of these messages like in line 27.

File: cmd/main.go [hunk 4]
```diff
@@ -1,42 +1,41 @@
...
 			}
 		}
 	}
 
+	//a.Run(ctx)
```
Comment (silverCase) [inline cmd/main.go:39]: What is this?
If this is here for convenience - leave a comment why it is there and for what purpose

File: internal/app/app.go [hunk 5]
```diff
@@ -1,23 +1,43 @@
...
-func NewApp() (*App, error) {
+func NewApp(cfg *config.Config) (*App, error) {
+	logF, err := logging.InitLog(cfg)
+	if err != nil {
+		return nil, err
+	}
 	mdb, err := mongodb.InitMongo()
-	bApi, bUpdateChannel := telegramapi.InitBot()
+	if err != nil {
+		return nil, err
```
Comment (silverCase) [inline internal/app/app.go:30]: codestyle
remove redundant tab

File: internal/app/app.go [hunk 6]
```diff
@@ -1,23 +1,43 @@
...
+	}
+
+	bApi, mesChan, err := telegramapi.InitBot(cfg)
+	if err != nil {
+		return nil, err
```
Comment (silverCase) [inline internal/app/app.go:35]: codestyle
remove redundant tab

File: internal/app/app.go [hunk 7]
```diff
@@ -1,23 +1,43 @@
...
+	}
+	log.Printf("Authorized on account %s", bApi.Self.UserName)
 
 	return &App{Mdb: mdb,
-		Bapi:           bApi,
-		BUpdateChannel: bUpdateChannel}, err
+		Bot:      bApi,
+		MessChan: mesChan,
+		LogFile:  logF}, err
```
Comment (silverCase) [inline internal/app/app.go:42]: codestyle
change positions of arguments: `LogFile:  logF}, err` is very confusing, `{` and `}` should not blend with arguments

File: pkg/config/config.go [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+	return &Config{
+		TeleToken: getVarEnv("TELEGRAM_TOKEN", ""),
+		Debug:     boolEnv(getVarEnv("DEBUG", "true")),
+		LogPath:   getVarEnv("LOG_PATH", "./log.txt"),
+		LogLevel:  getVarEnv("LOG_LEVEL", "info"),
+	}, nil
+}
+
+// Simple helper function to read an environment or return a default value
```
Comment (silverCase) [inline pkg/config/config.go:31]: to read an environment variable*

**Review by silverCase — CHANGES_REQUESTED** [review]
fix all comments

File: cmd/main.go (hunk 1 above)
Comment (yaroslavvasilenko) [reply cmd/main.go:18]: done

File: cmd/main.go (hunk 3 above)
Comment (yaroslavvasilenko) [reply cmd/main.go:34]: Done

File: cmd/main.go (hunk 4 above)
Comment (yaroslavvasilenko) [reply cmd/main.go:39]: this is future. I deleted

File: internal/app/app.go (hunk 6 above)
Comment (yaroslavvasilenko) [reply internal/app/app.go:35]: No

File: internal/app/app.go (hunk 7 above)
Comment (yaroslavvasilenko) [reply internal/app/app.go:42]: Ok

**Pull request merged**